Northstar.CustomServers/mod/scripts/vscripts/mp/spawn.nut

```diff
@@ -261,10 +261,15 @@ bool function IsSpawnpointValid( entity spawnpoint, int team )
 			return false
 	}
 
-	array<entity> projectiles = GetProjectileArrayEx( "any", TEAM_ANY, TEAM_ANY, spawnpoint.GetOrigin(), 600 )
-	foreach ( entity projectile in projectiles )
+    const minEnemyDist = 1000.0 // about 20 meters?
+	foreach ( entity projectile in GetProjectileArrayEx( "any", TEAM_ANY, TEAM_ANY, spawnpoint.GetOrigin() )
```

Comment:
```suggestion
	foreach ( entity projectile in GetProjectileArrayEx( "any", TEAM_ANY, TEAM_ANY, spawnpoint.GetOrigin(), minEnemyDist )
```
Shouldn't we check for distance to enemy projectiles as well?

Reply:
Yeah, good find